packages/keyring-controller/src/KeyringController.ts

```diff
@@ -364,6 +364,15 @@ export type ExportableKeyEncryptor = GenericEncryptor & {
   importKey: (key: string) => Promise<unknown>;
 };
 
+export type KeyringSelector =
+  | {
+      type: string;
+      index?: number;
```

Comment:
Is this `index` attribute here in case there's more than one keyring of the same type?

Reply:
Exactly! It defaults to `0` and it is used only in case the consumer uses a `type` to select the keyring